scripts/check_formatting.sh

```diff
@@ -5,6 +5,10 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 source $DIR/_common.sh
 
+exec yarn run \
```

Comment:
Out of curiosity, why are we using `exec`? Also probably just want to either remove the `exec docker buildx` below or at least comment it out.

Reply:
done